src/cr/cube/cubepart.py

```diff
@@ -1624,16 +1610,17 @@ def row_labels(self):
         """1D str ndarray of name for each row, suitable for use as row headings."""
         return self._assembler.row_labels
 
-    @lazyproperty
-    def row_order(self):
-        """1D np.int64 ndarray of signed int idx for each assembled row of stripe.
+    def row_order(self, format=ORDER_FORMAT.NEGATIVE_INDEXES):
```

Comment:
You know better than me how its used in exporter, do we need to find a way to cache?